app/js/actions/plugins.js

```diff
@@ -20,7 +64,8 @@ export const loadWorkflows = () => {
             .then((response) => (response.json()))
             .then((json) => {
                 Object.keys(json.workflows).map(workflow =>
-                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow]))
+                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow])) &&
```

Comment:
using `forEach` will avoid that lint error.

On Fri, Jun 17, 2016 at 4:49 PM, Jorden Kreps <notifications@github.com>
wrote:

> In app/js/actions/plugins.js
> <https://github.com/qiime2/qiime-studio/pull/37#discussion_r67589144>:
>
> > @@ -20,7 +64,8 @@ export const loadWorkflows = () => {
> >              .then((response) => (response.json()))
> >              .then((json) => {
> >                  Object.keys(json.workflows).map(workflow =>
> > -                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow]))
> > +                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow])) &&
>
> Yeah, that was an eslint issue, where
>
> *Error (array-callback-return)
> <http://eslint.org/docs/rules/array-callback-return> Expected to return a
> value in this function. at line ...*
>
> I'll switch it to just return nothing.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/qiime2/qiime-studio/pull/37/files/07a50f53adfa45ea46eee0d6511438e5ab0b781e#r67589144>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADyuZMJqLg9Qcfb7pYJ7sLLQEGJG2V_Jks5qMzKMgaJpZM4I1xjt>
> .
>